tests/src/actionContainers/SwiftActionContainerTests.scala

```diff
@@ -196,7 +198,29 @@ class SwiftActionContainerTests extends BasicActionRunnerTests with WskActorSyst
         })
     }
 
-        it should "properly use KituraNet and Dispatch" in {
+    it should "support pre-compiled binary in a zip file" in {
```

Comment:
Given how Swift isn't ABI stable, there's a risk that if you included a one-off zip file, it would have to be re-created whenever the Swift compiler version is updated. 

This could be a bit brittle for a test of this feature?